### ISSUE TITLE
default export fails to build with create-react-app

### ISSUE THREAD

**Issue by quentin-sommer** [developit/workerize#12]
When importing `workerize` in a cra app the build fail with this message

```
> react-scripts build

Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/workerize/src/index.js:23

Read more here: http://bit.ly/2tRViJ9
```

I was able to make it work using `import workerize from 'workerize/dist/workerize.m.js'`

**Comment by developit** [comment]
Whoops, the `"module"` entry is wrong.

**Comment by developit** [comment]
Fixed in 0.1.3!
